release notes/v0.36.0.md

```diff
@@ -0,0 +1,136 @@
+k6 v0.36.0 is here! 🎉 It introduces a couple of new features which enhance its usability, includes a number of fixes, and the result of ongoing refactoring efforts.
+
+## New Features!
+
+### Source Maps support ([#2082](https://github.com/grafana/k6/pull/2082))
+
+Following [#2082](https://github.com/grafana/k6/pull/2082), k6 now has support for [Source Maps](https://developer.mozilla.org/en-US/docs/Tools/Debugger/How_to/Use_a_source_map). k6 will try to load source maps either from the filesystem(s) or from inside the script, based on the standard(-ish) `//#sourceMappingURL=` comments. Furthermore, as k6 internally uses Babel to transform ES6+ scripts to ES5.1+, it will now make use of its ability to generate source maps, including combining with previously generated ones, to report correct line numbers. This should fix [#1804](https://github.com/grafana/k6/issues/1804); however, we anticipate some more issues will arise, and further tweaking will be necessary.
+
+Thus, given an `imported.js` module such as:
+```javascript
+export function f1() {
+  throw "line 2";
+}
+
+export function f2() {
+  throw "line 6";
+}
+
+export function f3() {
+  throw "line 10";
+}
+```
+
+and a k6 test script importing it as such:
+```javascript
+import { f2 } from "./imported.js"
+
+export default function() {
+  f2();
+}
+```
+
+Previous versions of k6 would report an error stacktrace indicating an invalid line number in `imported.js` (*10*):
+```
+ERRO[0000] line 6
+        at f2 (file:///some/path/imported.js:10:61(2))
+        at file:///some/path/sourcemap.js:4:20(4)  executor=per-vu-iterations scenario=default source=stacktrace
+```
+
+Starting with `v0.36.0` and source maps support, k6 would now report the exception at the correct line in `imported.js`:  
+```
+ERRO[0000] line 6
+        at f2 (file:///some/path/imported.js:6:2(2))
+        at file:///some/path/loadtest.js:4:2(4)
+        at native  executor=per-vu-iterations scenario=default source=stacktrace
+```
+
+#### Temporary caveat
+
+- If the set of scripts imported by K6 is bigger  
```

Comment:
Something is missing here :sweat_smile:

Reply:
Hehe indeed. I started writing it, only to realize I was sure what I should write, and decided to ask @yorugac [instead](https://github.com/grafana/k6/pull/2340#discussion_r787663394) 😸